asctb-api/src/index.ts

```diff
@@ -116,6 +116,30 @@ app.get('/v2/:sheetid/:gid', async (req: any, res: any) => {
   }
 });
 
+app.post('/v2/getDataFromCSV', async (req: any, res: any) => {
```

Comment:
Change route should be `/v2/csv`. All lowercase (and dashes) is a best practice for routes.

Reply:
sure will do that